app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,37 @@
+class MenusController < ApplicationController
+  def index
+    @menus = Menu.all
+    authorize @menus
```

Comment:
`authorize` before action. You have to check first if you have permissions to do something and then act